pkg/deployment/reconcile/plan_executor.go

```diff
@@ -397,10 +397,14 @@ func (d *Reconciler) executeAction(ctx context.Context, planAction api.Action, a
 		d.context.CreateEvent(k8sutil.NewPlanAbortedEvent(d.context.GetAPIObject(), string(planAction.Type), planAction.MemberID, planAction.Group.AsRole()))
 		return false, true, false, false, nil
 	} else if isActionTimeout(timeout, planAction) {
+		if planAction.Type.Optional() {
+			log.Warn("Optional action not finished in time. Skipping")
+			return false, false, false, false, nil
```

Comment:
Mark as ready, now Operator will just think action did not finish

Reply:
Done